src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReaderServiceImplTest {
+    private ReaderService readerService;
+
+    @BeforeEach
+    void setUp() throws IOException {
+        readerService = new ReaderServiceImpl();
+    }
+
+    @Test
+    void fileNotFound() {
+        Exception exception = assertThrows(RuntimeException.class,
+                () -> readerService.readFromFilesContents("src/main/resources/missing_file.txt"));
+        String expectedMessage = "Can't read from the file: src/main/resources/missing_file.txt";
```

Comment:
When you find yourself reusing a string, it's always a good idea to extract it to a constant. This way, you will only ever need to make changes in one place, not multiple places.